legend-engine-server/src/test/resources/org/finos/legend/engine/server/test/redshift.properties

```diff
@@ -0,0 +1 @@
+password = password
```

Comment:
Do we need this?

Reply:
Actually we don't need it when pushed. I have added it to the .gitignore file